data-table/table.js

```diff
@@ -157,6 +157,42 @@ function addColumns(numberOfColumns, index) {
      */
 }
 
+/**
+ * This function takes the id of the list container for the column names
+ * It then adds to the list another space for another column at the bottom of the list
+ * @param config id of the table
+ */
+function addColumn(config) {
+    let container = document.getElementById(config.tableIds.entryBoxDivId);
+    let child = document.createElement("DIV");
+
+    child.innerHTML = "Enter "+ config.datumConfig.names + ":";
```

Comment:
`config.datumConfig.names` is actually an array! Might be a good idea to change it to a dictionary, now that I'm thinking about it. Anyways, for now, any call to that particular field should look like `config.datumConfig.names[0]`